python/cudf/cudf/core/dtypes.py

```diff
@@ -262,7 +262,7 @@ def _init_categories(
             getattr(categories, "dtype", None),
             (cudf.IntervalDtype, pd.IntervalDtype),
         ):
-            dtype = "object"  # type: Any
+            dtype = CUDF_STRING_DTYPE
```

Comment:
So is this the same as `np.dtype("object")`?

Reply:
Correct. Trying to use this constant more throughout cudf Python just in case we ever change the Python representation of "string" type.

Reply:
Cool thanks